packages/next/telemetry/events/version.ts

```diff
@@ -97,6 +99,9 @@ export function eventCliSession(
     localeDomainsCount: i18n?.domains ? i18n.domains.length : null,
     localeDetectionEnabled: !i18n ? null : i18n.localeDetection !== false,
     imageDomainsCount: images?.domains ? images.domains.length : null,
+    imageRemotePatternsCount: nextConfig?.experimental?.images?.remotePatterns
+      ? nextConfig.experimental.images.remotePatterns.length
+      : null,
```

Comment:
```suggestion
    imageRemotePatternsCount: experimental?.images?.remotePatterns?.length ?? null
```
Nit: we could destructure `experimental` from `nextConfig` above like `images` and `i18n`

Reply:
Sounds good! Fixed in 55679a3